### PR TITLE
Docs: Compatibility page review

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR:
- Adds review suggestions for the new Compatibility page (added in #1048)

### CODE REVIEW

**Comment by kirszenbaum** [comment]
@sequba - this PR contains my ideas for the compatibility content

- As you said we're considering adding some forms of integration in the future, I went for the "compatibility" term after all
- The **Separators** section basically says consistency with Excel is not possible, so I moved it lower (not to start with bad news)
- I know adding the Google Sheets page changes the scope, but we can fill in  more details later, using the same pattern as on the Excel page (in the meantine, I added what I could check on my own)

File: docs/guide/compatibility-with-google-sheets.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- [`arrayColumnSeparator`](../api/interfaces/configparams.md#arraycolumnseparator)
+
+### Date and time formats
+
+In Google Sheets, date and time formats depend on your configured locale, whereas in HyperFormula, you set up date and time formats through these options:
+- [`dateFormats`](../api/interfaces/configparams.md#dateformats)
+- [`timeFormats`](../api/interfaces/configparams.md#timeformats)
+- [`nullYear`](../api/interfaces/configparams.md#nullyear)
+
+To set up HyperFormula in the same way as Excel's `en-US` locale, use this configuration:
```
Comment (sequba) [inline docs/guide/compatibility-with-google-sheets.md:74]:
```suggestion
To set up HyperFormula in the same way as Google Sheet's `en-US` locale, use this configuration:
```

**Review by sequba — CHANGES_REQUESTED** [review]

**Comment by sequba** [comment]
I like the idea of separate pages for "Compatibility with Microsoft Excel" and "Compatibility with Google Sheets". @warpech WDYT?

File: docs/guide/compatibility-with-google-sheets.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+### Array arithmetic mode
+
+In Google Sheets, the [array arithmetic mode](arrays.md#array-arithmetic-mode) is enabled by default.
+
+To set up HyperFormula in the same way, set the [`useArrayArithmetic`](../api/interfaces/configparams.md#usearrayarithmetic) option to `true`.
+
+```js
+useArrayArithmetic: true,
+```
```
Comment (sequba) [inline docs/guide/compatibility-with-google-sheets.md:26]:
```suggestion
In Google Sheets, the [array arithmetic mode](arrays.md#array-arithmetic-mode) is disabled by default.

To set up HyperFormula in the same way, set the [`useArrayArithmetic`](../api/interfaces/configparams.md#usearrayarithmetic) option to `false`.

```js
useArrayArithmetic: false, // set by default
```
```

File: docs/guide/compatibility-with-google-sheets.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+thousandSeparator: '', // set by default
+arrayColumnSeparator: ',', // set by default
+arrayRowSeparator: ';', // set by default
+dateFormats: ['MM/DD/YYYY', 'MM/DD/YY', 'YYYY/MM/DD'],
+timeFormats: ['hh:mm', 'hh:mm:ss.sss'], // set by default
+nullYear: 30, // set by default
+localeLang: 'en', // set by default
+useArrayArithmetic: true,
+leapYear1900: false, // set by default
+smartRounding: true, // set by default
```
Comment (sequba) [inline docs/guide/compatibility-with-google-sheets.md:103]: Syntax highlighting looks better this way

```suggestion
const options = {
  functionArgSeparator: ',', // set by default
  decimalSeparator: '.', // set by default
  thousandSeparator: '', // set by default
  arrayColumnSeparator: ',', // set by default
  arrayRowSeparator: ';', // set by default
  dateFormats: ['MM/DD/YYYY', 'MM/DD/YY', 'YYYY/MM/DD'],
  timeFormats: ['hh:mm', 'hh:mm:ss.sss'], // set by default
  nullYear: 30, // set by default
  localeLang: 'en', // set by default
  useArrayArithmetic: true,
  leapYear1900: false, // set by default
  smartRounding: true, // set by default
};
```

**Review by sequba — CHANGES_REQUESTED** [review]

**Review by sequba — APPROVED** [review]
Thanks @kirszenbaum. I'm merging it.

**Pull request merged**